cypress/e2e/1-getting-started/todo.cy.js

```diff
@@ -0,0 +1,143 @@
+/// <reference types="cypress" />
```

Comment:
@openai The purpose of this pull request is to contribute workflow/environment tooling and test coverage to an existing vanilla JS social media application. These tests are provided by Cypress during init. Should they be removed from the PR?